cpu/imxrt117x-cm4/lib/_startc.c

```diff
@@ -0,0 +1,41 @@
+/*
+ * Phoenix-RTOS
+ *
+ * i.MX RT117x Cortex-M4 core start code
+ *
+ * Copyright 2021 Phoenix Systems
+ * Author: Aleksander Kaminski
+ *
+ * This file is part of Phoenix-RTOS.
+ *
+ * %LICENSE%
+ */
+
+#include <stddef.h>
+
+#include "cm4.h"
+#include "gpio.h"
+#include "string.h"
+
+extern char __bss_start;
+extern char _end;
+extern int main(int argc, char *argv[]);
+
+
+__attribute__((noreturn)) void _startc(void)
+{
+	memset(&__bss_start, 0, &_end - &__bss_start);
```

Comment:
`__bss_start` points to the end of data section. If you will define `.data` in different location than `.bss`, this operation will zero additional area. 
The same case is consider in plo and additional variable was added: https://github.com/phoenix-rtos/plo/blob/706c50e2ca2cc4f9e760d470faf5cc03a9a8736f/_startc.c#L34

Reply:
I will add a linker script as it will be needed to be able to initialize .data section. This code will be then reworked

Reply:
ok